### PR TITLE
feat(grid): add support for small column width

### PR DESCRIPTION
**Pull request by hdimitrieski**

### CODE REVIEW

File: projects/data-grid-demo/src/app/app.component.ts [hunk 1]
```diff
@@ -100,12 +100,9 @@ export class AppComponent implements OnInit {
       field: 'age',
       component: NumberComponent,
       visible: true,
-      sortAvailable: true,
-      filterAvailable: true,
-      filter: {
```
Comment (velinovskav) [inline projects/data-grid-demo/src/app/app.component.ts:105]: Is this removed by mistake?

Comment (hdimitrieski) [reply projects/data-grid-demo/src/app/app.component.ts:105]: No. It's removed in order to show small column without sort and filter buttons.

File: projects/data-grid/src/lib/util/columns-style.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ export interface ColumnsStyle {
 
 const minmaxStyle = (width: string): string => `minmax(150px, ${width})`;
 
-const MIN_ALLOWED_WIDTH = 200;
+const MIN_ALLOWED_WIDTH = 0;
```
Comment (ignatij) [inline projects/data-grid/src/lib/util/columns-style.ts]: The min allowed width is 0 or 1 ?

Comment (hdimitrieski) [reply projects/data-grid/src/lib/util/columns-style.ts]: It's 1. If you enter 0 or less than 0, it won't take the value into an account.

Comment (hdimitrieski) [reply projects/data-grid/src/lib/util/columns-style.ts]: I will change it to 1 for clarity.

**Review by velinovskav — APPROVED** [review]

**Pull request merged**